server/middleware/error.js

```diff
@@ -8,6 +8,7 @@ export default function() {
       await next();
       if (404 === ctx.response.status && !ctx.response.body) ctx.throw(404);
     } catch (err) {
+      console.log(err);
```

Comment:
I've put this in as a `console.error` on my PR.
https://github.com/wellcometrust/wellcomecollection.org/pull/1179/commits/5af4c56bf763f52ad4991b4595c24e272d8c6289#diff-7060d422a62e54a916f3f499727cda56R16